Assets/Scripts/StandardSamples/OpenID/OpenIdTokenManager.cs

```diff
@@ -20,6 +20,8 @@
 * SOFTWARE.
 */
 
+using Epic.OnlineServices;
+using Epic.OnlineServices.Connect;
 using System;
 using System.Collections;
 using System.Collections.Generic;
```

Comment:
You had previously (and rightly in my mind) expressed a preference for placing `using` statements within `namespace` code blocks. This would be a good opportunity to make that change at least in this file.

Reply:
Resolved by moving the statements into the namespace.